Makefile

```diff
@@ -127,7 +127,7 @@ gas: $(GAS)
 # convenience targets
 all: components tethers isos vic-machine vic-ui
 tools: $(GOIMPORTS) $(GVT) $(GOLINT) $(SWAGGER) $(GAS) goversion
-check: goversion goimports govet golint copyright whitespace gas
+check: goversion goimports govet lintcode copyright whitespace gas
```

Comment:
Is the `lintcode` binary in the standard package?  If not, it'll need to be added to vendor, and appropriate `install` command added to the makefile.